bagua/script/baguarun.py

```diff
@@ -139,13 +150,23 @@ def parse_args():
 
     if args.host_list is None:
         args.host_list = os.environ.get("BAGUA_NODE_DOMAIN_NAMES", "")
-    if args.ssh_port is None:
+    if args.ssh_port is None and "BAGUA_SSH_PORT" in os.environ:
```

Comment:
What is `BAGUA_SSH_PORT`? KML internal variable?

Do we need to document this

Reply:
Yes, explained in argparse help.